index.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css"
+    />
+    <link rel="stylesheet" href="./index.css"/>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato:wght@100&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200&display=swap" rel="stylesheet">
+
+</head>
+  <body>
+    <header>
+     <nav> 
+    <img src="./Images/Hamburger_icon.svg.png" class="hamburger" alt="" />
+      <ul class="menu-items">
+        <img src="./Images/Cross-Icon.png" class="Cross-Icon" alt="" />
+        <li class="menu-item-mobile">
+          <a href="./about.html">About</a>
+        </li>
+        <li class="menu-item-mobile">
+          <a href="#">Program</a>
+        </li>
+        <li class="menu-item-mobile">
+          <a href="#">Join</a>
+        </li>
+        <li class="menu-item-mobile">
+          <a href="#">Sponsor</a>
+        </li>
+        <li class="menu-item-mobile">
+          <a href="#">Help</a>
+        </li>
+      </ul>
+
+      <div class="myLinks">
+        <ul>
+          <li><a href="www.facebook.com"></a> <img src="/Images/fb.icon.png" alt=""></li>
+          <li><a href="www.gooogle.com"></a><img src="/Images/Twttr-logo.png" alt=""></li>
+          <li><a href="/index.html">Login</a></li>
+          <li><a href="about.html">MyPage</a></li>
+          <li><a href="#">English</a></li>
+        </ul>
+      </div> 
+      
+      <ul class="nav-list">
+        <li class="menu-item">
+          <img src="./Images/My-Logo.png" alt="Logo">
+        </li>
+        <li class="menu-item">
+          <a href="./about.html">About</a>
+        </li>
+        <li class="menu-item">
+          <a href="#">Program</a>
+        </li>
+        <li class="menu-item">
+          <a href="#">Join</a>
+        </li>
+        <li class="menu-item">
+          <a href="#">Sponsor</a>
+        </li>
```

Comment:
- [ ] Please from the project requirements, you should stick to the [Project design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) as much as possible. But as you can see you have some differences in the design. 

You can see the differences below

- [ ] Good job :+1:  adding the `top nav bar` :100: , Please make sure the top nav bar follows the design. 
      - The links should have a `margin-right` to match the design
      - the top nav should have the Facebook and Twitter icons
      - your links should be in other of `English, My Page, Logout` instead of `Login, My Page, English`
      - links shouldn't be underlined.

- [ ] As requested by the previous reviewer, please make sure that your `main nav bar` follow the design. 

#### Your Design :point_down: 
![Screenshot from 2022-11-26 11-00-41](https://user-images.githubusercontent.com/92303891/204085445-63c1d373-a829-4aab-95a9-6a2efcaf74cc.png)
#### Project Design :point_down: 
 ![Screenshot from 2022-11-26 11-04-16](https://user-images.githubusercontent.com/92303891/204085519-0679fc3c-b8d3-4012-bc55-3f4e0f67a645.png)|